experiments/AMIP/modular/components/push_pull.jl

```diff
@@ -27,12 +27,11 @@ function land_pull!(cs)
     land_mask = cs.surface_masks.land
     parent(land_sim.integrator.p.bucket.ρ_sfc) .= parent(csf.ρ_sfc)
     parent(land_sim.integrator.p.bucket.turbulent_energy_flux) .=
-        apply_mask.(parent(land_mask), >, parent(csf.F_A), parent(csf.F_A) .* FT(0), FT(0))
+        parent(binary_mask.(land_mask, threshold = eps()) .* csf.F_A)
```

Comment:
We could call `binary_mask` using `Regridder.binary_mask` to make it more clear where it comes from, since it's not otherwise explicitly specified in this file.